spec/openapi.yaml

```diff
@@ -1859,6 +1859,18 @@ paths:
           schema:
             type: boolean
           example: true
+        - name: expiry_status
+          in: query
+          schema:
+            type: string
+            enum:
+              - EXPIRED
+              - EXPIRING
+          description: |
+            Filter sender addresses based on their expiry status.
+            This parameter is effective when used in MON request filter,
```

Comment:
> This parameter is effective when used in MON request filter,  where the sender address type is INTERNATIONAL and the usage type is OWN_NUMBER.

Best to reword this. Customer will not have an idea what `MON` is.

Reply:
thank you @archwin-sinch , let me update